src/pages/EditMonitor.vue

```diff
@@ -170,6 +170,15 @@
                                 <input id="retry-interval" v-model="monitor.retryInterval" type="number" class="form-control" required min="20" step="1">
                             </div>
 
+                            <div class="my-3">
+                                <label for="resend-interval" class="form-label">
+                                    {{ $t("Notification resend Interval if Down") }}
```

Comment:
Should a translation have been added for this in `en.js`?
```suggestion
                                    {{ $t("Notification resend interval if down") }}
```

Reply:
Yes, I've missed that. Thanks 👍